docs/report.md

```diff
@@ -4,6 +4,8 @@ We anaylsis 621,377,878 records of GitHub logs, there are 39,759,065 active repo
 
 ## GitHub Word Cloud
 
+We select topics of top 10w active repositories, filtered the topics with a frequency of more than 25. Then a word cloud map is generated based on the data. The word cloud visually highlights the topics that appear frequently, so that we can quickly get the more popular topics.
```

Comment:
This file should be updated by robot, do not modify this one.

Reply:
OK, I will fix it.